ui/src/shared/components/ClickOutsideInput.tsx

```diff
@@ -5,6 +5,7 @@ import {ErrorHandling} from 'src/shared/decorators/errors'
 
 interface Props {
   min?: string
+  max?: string
```

Comment:
does this need a default value?

Reply:
no it doesn't have a max if it is not set.